firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/AabUpdater.java

```diff
@@ -48,18 +50,22 @@
   private AppDistributionReleaseInternal aabReleaseInProgress;
 
   private final Object updateAabLock = new Object();
+  private final Context applicationContext;
 
-  AabUpdater() {
+  AabUpdater(@NonNull FirebaseApp firebaseApp) {
```

Comment:
This constructor should just take a Context object since that's all you need.

Reply:
Good thought! I had done this elsewhere before should do it here as well.